islandoraUtils/DSConverter.py

```diff
@@ -59,6 +59,13 @@ def create_thumbnail(obj, dsid, tnid):
         if r == 0:
             r = subprocess.call(['convert', '%s[0]' % tmpfile, '-thumbnail', '%sx%s' % tn_size,\
                  '-colorspace', 'rgb', 'jpg:%s'%tnfile])
+    # special case for pdfs
+    # flatten the layers to avoid issues with transparencies and resizing,
+    # which can result in black thumbnails
+    elif mime == 'application/pdf':
+        r = subprocess.call(['convert', '%s[0]' % infile, '-thumbnail', \
+             '%sx%s' % tn_size, '-colorspace', 'rgb', '-flatten', 'jpg:%s'%tnfile])
```

Comment:
Almost thinking it may make more sense to just add the `-flatten` to the base case below... Could hypothetically have `image/png` or other formats which support multiple layers/transparency exhibit the same behaviour?